zeekpkg/template.py

```diff
@@ -483,7 +483,7 @@ def contentdir(self):
         Returns:
             str: relative path to the content directory
         """
-        return None
+        pass
```

Comment:
```suggestion
        ...
```

Seems the current Python docs use ellipsis for the placeholder: https://docs.python.org/3/library/abc.html#abc.abstractmethod

Reply:
I'd prefer to not do that change now since nothing here says that this is an abstract method and ellipsis seem to be mostly used to denote a placeholder (e.g., to declare signatures `fn foo(): ...`). If we cleaned up `template.py` to declare proper abstract interfaces we should revisit that.

Reply:
Also resolving this one. Feel free to merge, seems you and Evan think this is worth it, so :+1: